styles.css

```diff
@@ -0,0 +1,427 @@
+.header {
+  height: 330px;
```

Comment:
You should be using %, rem/em instead of px for most css from now on. px are not responsive.

Reply:
Adjust all the px to rem or em.